docs/source/pillars/computing_technology.md

```diff
@@ -59,39 +59,67 @@ _The tools used by researchers inside a TRE, such as programming languages, IDEs
 * - Statement
   - Guidance
   - Importance
-* - Your TRE must provide software applications that are relevant to working with the data in the TRE.
-  - The tools provided will depend on the types of data in the TRE, and the expectations of users of the TRE.
-    For users working in a TRE via a virtual desktop, this may include programming languages such as Python and R, integrated development environments, Jupyter notebooks, office type applications such as word processors and spreadsheets, command line tools, etc.
-    TREs with non-desktop interfaces should similarly consider carefully which applications are best suited for the researchers needs when interacting with the data, for example "point and click" GUI tools for querying a database and generating plots of data.
-    The set of tools should be reviewed regularly to ensure they are up to date.
-  - Mandatory
```

Comment:
Does the specification need an appendix where we could put examples of TRE components? This could read something like:

- Your TRE must provide software applications that are relevant to working with the data in the TRE.
- The set of tools provided will depend on the types of data in the TRE, and the expectations of the TRE users, and should be reviewed regularly to ensure they are up to date. For examples of the kinds of software applications used by existing TREs, refer to the [appendix] <- *link to relevant part*
- Mandatory

Reply:
Where things have been removed, I'm not completely sure why. I think it's fine to say that we choose to keep them (and amend the WP3 document accordingly). @JimMadge: do you want to reword this to be a bit less verbose?

Reply:
Reverted in be1f53a